frontend/src/tests/import-export.test.js

```diff
@@ -2,47 +2,173 @@
  * @jest-environment node
  */
 import { describe, it, expect } from "./utils";
-import { validate, SpreadsheetRowMapper } from "../libs/importExportUtils";
+import {
+    validate,
+    SpreadsheetRowMapper,
+    normalizeImport,
+} from "../libs/importExportUtils";
+import { prepareMinimal } from "../libs/exportUtils";
+import XLSX from "xlsx";
+import {
+    objectJSON,
+    instructorSchema,
+    applicantSchema,
+    positionSchema,
+    assignmentSchema,
+    ddahSchema,
+    instructorData,
+    applicantData,
+    positionData,
+    assignmentData,
+    ddahData,
+} from "./test-data-file/test_data";
```

Comment:
Is this directory a file? If not, it shouldn't have the word `file` in its name.